src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1087,20 +1089,25 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
                         {
                             context.Visitor.Visit(member.Expression);
                             writer.Write(".");
-
                         }
 
                         var name = member.Member.Name;
-                        if (ReservedWordsSupport.JsReservedWords.Contains(name))
+
+                        if (DoNotIgnore && name.StartsWith(TransparentIdentifier))
+                        {
+                            name = name.Substring(2);
```

Comment:
its' `<>` right? let's put a comment or better move it to const and use .Length here - it will make it easier to figure out what's going on here in the future

Reply:
done